<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Connections</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
			    background-color: rgb(255, 255, 255);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
              }
QLabel {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QLineEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QGroupBox {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
              }</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>378</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
               color: white;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
               max-width: 120px;
               background-color: rgb(0, 85, 255);
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: darkblue;
               border: 1px solid grey;
               background-color: rgb(255, 255, 255);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
			    background-color: rgb(0, 78, 235);
               }</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QToolButton" name="rejectButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
    </property>
    <property name="text">
     <string>Reject</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Host name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
